packages/report-server/src/reportBuilder/output/output.ts

```diff
@@ -14,13 +14,25 @@ type OutputParams = {
 const output = (rows: Row[], params: OutputParams) => {
   const { type, config } = params;
 
-  const outputBuilder = outputBuilders[type as keyof typeof outputBuilders](config);
+  const outputBuilder = outputBuilders[type](config);
   return outputBuilder(rows);
 };
 
 const buildParams = (params: unknown): OutputParams => {
+  const isOutPutBuildersType = (value: string): value is keyof typeof outputBuilders => {
+    switch (value) {
+      case 'default':
+      case 'matrix':
+        return true;
+      default:
+        return false;
+    }
+  };
```

Comment:
Find a hacky way to deal with the object key types verification. Not sure this is a good practice?